### ISSUE TITLE
No installs result in runnable software

### ISSUE THREAD

**Issue by lnxchk** [chef/okta_aws#12]
I have tried all of the install methods documented in the README, and none of them install a script in a PATH directory.

Is it expected that we have to download the git repo and run the software from there?

I'm on MacOS 10.13.5, and upgraded to python3.7 both manually and via brew to attempt to get this software to work. I also updated pip, and tried `alias python='python3'` to try to get the installs working.

**Comment by mivok** [comment]
If you're on a mac, It's expected that `brew install okta_aws` (once you have tapped chef/okta_aws) should work. You shouldn't have to download the git repo at all unless you want to do development on okta_aws. Can you paste any errors you are getting or, if there aren't any, can you paste or send a copy of the results of running brew install please?

**Comment by jtimberman** [comment]
We got @lnxchk sorted out. We first tried doing an install of `okta_aws`, and encountered a no suhc module error on setuptools, indicating a problem with Python (3) itself. After digging around, we discovered that it was trying to use the system Python (references to `/Library/Frameworks` stuff). We then performed the following:

```
brew reinstall python
```

Which resulted in brew thinking it would install from source rather than the precompiled bottle. This is likely due to the original installation being a source based install, predating when brew defaulted to precompiled packages.

We cancelled that, and then this worked instead:

```
brew link python
brew postinstall python
brew install okta_aws
```

And then `okta_aws` worked, signed into the configured AWS account, and `aws` CLI commands were usable.
